geometry_msgs/PoseStamped estimateArCamPose
geometry_msgs/PoseStamped trueArCamPose
geometry_msgs/PoseStamped estimateArOdomPose
geometry_msgs/PoseStamped trueArOdomPose
